src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
Very testable method, a test case would help you make sure you don't have any regressions in case the logic needs to change for whatever reason. Please consider adding  a test case for this new code.